README.md

```diff
@@ -209,7 +209,7 @@
 
 |필드|타입|설명|
 |---|---|---|
-|json array|List|- 게시글 번호 <br> - 제목<br> - 이름<br> - 작성일<br> - 조회수<br> 정보를 가진 객체의 List를 반환 합니다.|
+|needteamArticles|List|- 게시글 번호 int <br> - 제목 String<br> - 이름 String<br> - 작성일 Date<br> - 조회수 int<br> 정보를 가진 객체의 List를 반환 합니다.|
```

Comment:
Do not mark resolved for what we are discussing, not finished.
Please consider readability for readers.
https://github.com/dgryoo/pfairplayService/pull/2#discussion_r546374252